h/features/models.py

```diff
@@ -63,6 +80,10 @@ class Feature(Base):
                       default=False,
                       server_default=sa.sql.expression.false())
 
+    featurecohorts = sa.orm.relationship('FeatureCohort',
+                                         secondary=FEATURE_FEATURECOHORT_TABLE,
+                                         backref='cohorts')
```

Comment:
The `backref` property determines the name of the attribute on the *other side* of the relationship. So this would put a `cohorts` attribute on instances of the `FeatureCohort` class, which I don't imagine is what you intended.